src/main/kotlin/com/atlassian/performance/tools/dockerinfrastructure/jira/SetUpFromScratchAction.kt

```diff
@@ -59,22 +62,24 @@ internal class SetUpFromScratchAction(
     }
 
     /**
-     *  Uses "10 user starter non-eval host product license, expires in 3 hours" license
-     *  from https://developer.atlassian.com/platform/marketplace/timebomb-licenses-for-testing-server-apps/.
+     *  Uses "10 user Jira Software Data Center license, expires in 3 hours" license
```

Comment:
Should we have a JSM compatible license here? This requirement may not be tested by our automated tests, but I'm not sure if it's not a requirement already required.

Possibly the "3 hour expiration for all Atlassian host products*" from the https://developer.atlassian.com/platform/marketplace/timebomb-licenses-for-testing-server-apps/ could cover JSM.